packages/sdk/src/adapters/eth-bridge.ts

```diff
@@ -178,7 +178,10 @@ export class ETHBridgeAdapter extends StandardBridgeAdapter {
           amount,
           0, // L1 gas not required.
           '0x', // No data.
-          opts?.overrides || {}
+          {
+            ...omit(opts?.overrides || {}, 'value'),
```

Comment:
nit: the omit here is unnecessary it's being overwritten on line 183

Reply:
Going to not change this as to be extra sure no behavior is changed in the old sdk version

Reply:
This will need to be refactored once its only bedrock support anyways